ui/oopsyraidsy/oopsy_summary_list.ts

```diff
@@ -216,9 +216,11 @@ export class OopsySummaryList implements MistakeObserver {
   OnMistakeObj(timestamp: number, m: OopsyMistake): void {
     const iconClass = m.type;
     const blame = m.name ?? m.blame;
-    const blameText = blame ? `${ShortNamify(blame, this.options.PlayerNicks)}: ` : '';
+    const blameText = (blame !== undefined && blame.length > 0)
+      ? `${ShortNamify(blame, this.options.PlayerNicks)}: `
+      : '';
     const text = Translate(this.options.DisplayLanguage, m.text);
-    if (!text)
+    if (text === undefined || text.length === 0)
```

Comment:
```suggestion
    if (text === undefined)
```

`text` is coming from triggers and should never be an empty string.